### PR TITLE
New package: MarcielDegasperi.MDBAdmin version 2.5.11.830

### PR DESCRIPTION
**Pull request by comhon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/143000)

### CODE REVIEW

**Comment by comhon** [comment]
@microsoft-github-policy-service agree

**Comment by comhon** [comment]
I have tested and use the program everyday, however SmartScreen marks the program, since the installer package does not have a digital signature from the author.

**Comment by stephengillie** [comment]
Hi @comhon,

Could you add a PackageUrl? 

This label isn't from SmartScreen, but because the host is one of many which provides both direct hosting, and is also used for 3rd party rehosting. So the check requires that a human (such as myself) verify that this is the former and not the latter. This package passed SmartScreen and Defender tests, and is ready to merge once the source is verified. 

The PackageUrl will help me to determine that this is the developer's hosting. It should be a website for the software package, and should be a few clicks away from the InstallerUrl. Generally, anyone should be able to navigate from the PackageUrl to the InstallerUrl without authenticating or passing a paywall. 

Thanks in advance for helping me to approve your PR.

**Comment by comhon** [comment]
Hello, thanks for help with the PR.
The project itself is hosted on SourceForge.

I am not sure which of these links I shall use.

*https://sourceforge.net/projects/mdbadmin/

*https://sourceforge.net/projects/mdbadmin/files/

Regarding my last question, won't it be a problem, that in my test environment, SmartScreen popped up with a confirmation during the installation?

This is my first contribution to winget packages repository, so I am sorry for the inconveniences.

File: manifests/m/MarcielDegasperi/MDBAdmin/2.5.11.830/MarcielDegasperi.MDBAdmin.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MarcielDegasperi.MDBAdmin
+PackageVersion: 2.5.11.830
+PackageLocale: en-US
+Publisher: Marciel Degasperi
+PackageName: MDB Admin
```
Comment (stephengillie) [inline manifests/m/MarcielDegasperi/MDBAdmin/2.5.11.830/MarcielDegasperi.MDBAdmin.locale.en-US.yaml:8]:
```suggestion
PackageName: MDB Admin
PackageUrl: https://sourceforge.net/projects/mdbadmin/
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**